querybook/webapp/components/ListDataDoc/components/DataDocSchedule.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import cronstrue from 'cronstrue';
+
+export const DataDocSchedule: React.FunctionComponent<{
```

Comment:
name is duplicated, use something like FriendlyCronSchedule or HumanReadableCronSchedule